### ISSUE TITLE
Maybe missing c.Next()

### ISSUE THREAD

**Issue by erdian718** [gofiber/basicauth#5]
```
if cfg.Authorizer(user, pass) {
	c.Locals("username", user)
	c.Locals("password", pass)
	// Maybe missing c.Next()
	return
}
```

**Comment by thomasvvugt** [comment]
Hi @ofunc ,
Could you refer to the specific occurrence of your issue?
Thanks in advance! 👍🏼

**Comment by ghost** [comment]
Hello,

after upgrading to fiber v10, I'm running in an odd issue that I pinned down to basicauth and is most likely caused by this issue: When auth is succesful, an empty response with Status Code 200 is returned. The function that is set to handle the path is never called.

This is likely due to a missing call to `c.Next()` before the return in https://github.com/gofiber/basicauth/blob/master/main.go#L90. Without that call fiber thinks the request is "done" by the middleware, and doesn't go on to call the handling function.

A quick test to confirm that: Using a simple filter that always returns `true` triggers the `c.Next()` in https://github.com/gofiber/basicauth/blob/master/main.go#L69, and the handling function is called.

Thanks!

**Comment by Fenny** [comment]
You are right, we are missing `c.Next()` before we call `return`. I'm tagging a fix [v0.2.0](https://github.com/gofiber/basicauth/releases/tag/v0.2.0) 👍
